### ISSUE TITLE
Add Alert Schema

### ISSUE THREAD

**Issue by mjoneill** [uiowa/uiowa#1352]
- [ ] @mjoneill to provide reference json, explanation, answer questions, etc.
- [ ] Implement schema in cms

Reference material:
* https://schema.org/SpecialAnnouncement
* https://developers.google.com/search/docs/data-types/special-announcements. <--Covid specific info.  Not sure we need to do this....but it is worth noting that many of the elements here are directly applicable to us/our mission (school closures, etc...)

Recommended attributes:
* Name
* Text
* datePosted
* Expires (if appropriate)
* Category
* URL

Parent Story: #1290

**Comment by mjoneill** [comment]
[faq-json.txt](https://github.com/uiowa/uiowa/files/4471268/faq-json.txt)

Reference json...

**Comment by joewhitsitt** [comment]
Should this be incorporated into #1275?

**Comment by mjoneill** [comment]
@joewhitsitt Yes, we can incorporate it...  if the opportunity is there and we want to pull it.  If it's going to add more uncertainty about getting it to done, I would be fine with pulling into a future sprint.

**Comment by joewhitsitt** [comment]
It looks like Rick is close on 1275 so, let's just plan to follow up on this one.

**Comment by richardbporter** [comment]
Do we need this for launch?

**Comment by mjoneill** [comment]
No. Nice to have. Not urgent.  I'm fine if this comes later/future.  Might even be worth having more convo about it.
